src/java/org/jivesoftware/openfire/session/LocalSession.java

```diff
@@ -413,4 +429,27 @@ public String getCipherSuiteName() {
         }
         return "NONE";
     }
+
+    /**
+     * Enables stream management for session
+     * @param enable XEP-0198 <enable/> stanza
+     */
+    public void enableStreamMangement(Element enable) {
+
+    	// Do nothing if already enabled
+    	if(streamManager.isEnabled()) {
+    		return;

```

Comment:
Should we reset the counter(s) on subsequent <enable />? Spec seems a little ambiguous on this point.